src/nvim/api/extmark.c

```diff
@@ -1291,3 +1281,47 @@ Boolean nvim__win_del_ns(Window window, Integer ns_id, Error *err)
 
   return true;
 }
+
+/// EXPERIMENTAL: this API will change in the future.
+///
+/// Set some properties for namespace
+///
+/// @param ns_id Namespace
+/// @param opts Optional parameters to set:
+///           - scoped: (boolean) Make the namespace a scoped namespace
```

Comment:
I'm still [wondering](https://github.com/neovim/neovim/pull/27361#discussion_r1597710114) why the `opts.scoped` parameter is needed. If a namespace was assigned to windows/buffers/etc, it's scoped. Else it's not scoped.

Reply:
Then I'll just remote the option